src/graph/graphcontext.c

```diff
@@ -434,7 +434,7 @@ void GraphContext_RemoveSchema(GraphContext *gc, int schema_id, SchemaType t) {
 }
 
 const char *GraphContext_GetEdgeRelationType(const GraphContext *gc, Edge *e) {
-	int reltype_id = Graph_GetEdgeRelation(gc->g, e);
+	int reltype_id = e->relationID;
```

Comment:
Would you be willing to consider using: `EDGE_GET_RELATION_ID` ?
Or may this macro is no longer valid, if that's the case see if we can remove it...